src/components/Candidates/CandidatesJoinUs/CandidatesOpenPositions/OpenPositionCard/OpenPositionCard.js

```diff
@@ -11,9 +11,14 @@ class OpenPositionCard extends Component {
     this.position = props.position;
   }
 
+  clicked = () => {
+    var pos = this.position;
```

Comment:
`var` is never used, only `let` and `const`

```suggestion
    const pos = this.position;
```